docs/sdks/uid2-sdk-ref-mobile.md

```diff
@@ -0,0 +1,212 @@
+
+The **UID2Manager** singleton is the primary developer API for the UID2
+SDK. It is responsible for storing, refreshing, and retrieving UID2
+Identity.
+
+The behavior is a little different depending on the SDK that's in use:
+
+-   iOS: The UID2Manager is initialized automatically the first time it
+    is accessed and can be configured to support automatic or manual
+    refresh capabilities.
+
+-   Android: The UID2Manager must be initialized manually before it can
+    be used (see Android Initialization content below).
+
+### UID2Manager API
+
+This section includes the functions and variables that are part of the
+UID2Manager API.
+
+#### Functions
+
+The following functions are available as part of the UID2Manager API.
+
+##### setIdentity()
+
+Sets the UID2 Identity to be managed by the SDK.
+
+##### resetIdentity()
+
+Resets or removes the UID2 Identity currently being managed by the SDK.
+
+##### refreshIdentity()
+
+Manually refreshes the UID2 Identity being managed by the SDK.
+
+##### getAdvertisingToken()
+
+If the current UID2 Identity is valid, this function returns the
+advertising token.
+
+##### setAutomaticRefreshEnabled()
+
+Toggle for automatic refresh functionality.
+
+#### Variables
+
+The following variables are available as part of the UID2Manager API.
+
+##### identity
+
+The Identity variable stores and returns the current UID2Identity data
+object being managed by the SDK.
+
+##### identityStatus
+
+The identityStatus variable stores and returns the status of the current
+UID2 Identity being managed by the SDK
+
+### Android Initialization
+
+The Android implementation expects the singleton to be initialized
+before use. This does two things:
+
+-   It allows for easier access later.
+
+-   It allows the consuming application to potentially provide its own
+    network instance, responsible for making requests.
+
+The initialization can be done during the creation of the APPLICATION
+instance, as shown in the following example:
+
+```java
+class MyApplication : Application() {
+
+override fun onCreate() {
+
+super.onCreate()
+
+// Initialize the UID2Manager class. Use DefaultNetworkSession rather
+than providing our own
+
+// custom implementation. This can be done to allow wrapping something
+like OkHttp.
+
+UID2Manager.init(this.applicationContext)
+```
+
+## Code Samples
+
+The following code samples provide examples of performing specific
+activities relating to managing UID2.
+
+Set the initial UID2 Identity (iOS):
+
+```swift
+UID2Manager.shared.setIdentity(_ identity: UID2Identity)
+```
+Set the Initial UID2 Identity (Android):
+```java
+UID2Manager.getInstance().setIdentity(identity: UID2Identity)
+```
+Get the Advertising Token to pass to the Advertising SDK (iOS):
+```swift
+UID2Manager.shared.getAdvertisingToken()
+```
+Get the Advertising Token to pass to the Advertising SDK (Android):
+```java
+UID2Manager.getInstance().getAdvertisingToken()
+```
```

Comment:
@genwhittTTD just a question when you come back to format this later - maybe we should have a Android guide and an iOS guide and each guides contain the relevant info of UID2 mobile sdk + GMA/IMA PLugin guide, instead of mixing iOS/Anroid content together? (similar to how we have separate pages for each of UID2 C++/Java/Python/C# sdk

Reply:
Hi @sunnywu definitely yes on this!

Reply:
Noting that this was done.